projects/RabbitMQ.Client/RabbitMQ.Client.csproj

```diff
@@ -62,4 +63,7 @@
     <PackageReference Include="System.IO.Pipelines" Version="7.0.0" />
   </ItemGroup>
 
+  <ItemGroup Condition="$(TargetFramework) == 'netstandard2.0'">
+    <PackageReference Include="System.Diagnostics.DiagnosticSource" Version="7.0.2" />
```

Comment:
nit: any reason to require 7.0? Would it be possible to add 6.0?
The context: some environments like Azure Functions (and I believe AWS Lambda) could have a problem with newer versions of DiagnsoticSource. E.g. https://github.com/Azure/azure-functions-host/issues/7135.
Of course, there would be no OTel story there, but at least some custom solutions might work.

So keeping the dependency version as low as possible would be nice.

Reply:
OpenTelemetry .NET took a stand to use the latest stable always, irrespective of the fact that environments like AzureFunctions might see issues. It does not mean that this repo should follow them! If there is a need to use newer features (Like Metrics), then it *must* to use newest, otherwise, lowest is good enough in this repo.

OTel .NET:
https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/Directory.Packages.props#L46-L54

Reply:
The aim is to start pushing metrics through OTel as well through another PR once tracing is in place.